### PR TITLE
Add command to generate a tenant authorization token

### PR DESCRIPTION
**Pull request by elv-peter**
Useful for testing the authd HTTP API at least

### CODE REVIEW

**Review by elv-todd — APPROVED** [review]
this is super helpful, ty!

sample usage:
```
$ ./elv-live tenant_auth_token /tnt/wlt/iten4TXq2en3qtu3JREnE5tSLRf9zLod > /tmp/f ; tok=`grep Auth /tmp/f` ; path=`grep iten /tmp/f` ; curl -s -H "$tok" http://localhost:6546$path
```

File: utilities/EluvioLiveCli.js [hunk 1]
```diff
@@ -44,6 +44,24 @@ const Init = async ({debugLogging = false, asUrl}={}) => {
   await marketplace.Init({ debugLogging });
 };
 
+const CmdTenantAuthToken = async ({ argv }) => {
+  await Init({debugLogging: argv.verbose});
+
+  let ts = Date.now();
+  let message = argv.path_or_body + "?ts=" + ts
+  try {
+    let j = JSON.parse(argv.path_or_body)
```
Comment (elv-todd) [inline utilities/EluvioLiveCli.js:53]: cn you share a sample url or body?  say i want to do like

```
curl -s -H 'Authorization: Bearer ES256K_...' http://localhost:6546/tnt/wlt/itenYQbgk66W1BFEqWr95xPmHZEjmdF
```
for example.

Comment (elv-peter) [reply utilities/EluvioLiveCli.js:53]: Example output:
```sh
# URL path
$ ./elv-live tenant_auth_token "/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a"
/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a?ts=1688181635434
Authorization: Bearer ES256K_6Rbkcyi7vzYzvQN4HjG3hJknUmdi1uWavxH79JFopWv6ysAQsH8c3hE7RcPpLKi4Rvxe7es8pFr1Ht52SfFvCk95C

# Request body (empty JSON payload)
$  ./elv-live tenant_auth_token "{}"
{"ts":1688181686275}
Authorization: Bearer ES256K_DAsxBZYZFcPjjK1ZajwRtCSqViRkucRHgwu7f7hQv6ENLnTEikVtcBisQqjav9p18STU5Z5rMVU3km8WXvMQB7y8v
```

File: utilities/EluvioLiveCli.js [hunk 2]
```diff
@@ -44,6 +44,24 @@ const Init = async ({debugLogging = false, asUrl}={}) => {
...
+    j.ts = ts;
+    message = JSON.stringify(j)
+  } catch (e) {}
+
+  const { multiSig } = await elvlv.TenantSign({
+    message: message
+  });
+  console.log(message)
+  console.log(`Authorization: Bearer ${multiSig}`);
```
Comment (elv-serban) [inline utilities/EluvioLiveCli.js:62]: Can we return a YAML that can have message and token?  The other commands return YAML (which is both human readable-ish and command line parseable).

**Review by elv-serban — APPROVED** [review]

**Review by elv-serban — APPROVED** [review]

File: utilities/EluvioLiveCli.js (hunk 2 above)
Comment (elv-peter) [reply utilities/EluvioLiveCli.js]:
```
~/e/git/elv-live-js $ ./elv-live tenant_auth_token '{"hello":123}'
Network: demov3
Timestamped path or body: {"hello":123,"ts":1689024549073}
Token: Authorization: Bearer ES256K_LtRWGcHG3y9EiRpMu7YKU8doYXmSSXPGgGfpbUjq8PKyMdXrD9Ngn9HhoxARMhovogzG4FnUvwapRyQSZMmzDrM2B
~/e/git/elv-live-js $ ./elv-live tenant_auth_token "/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a"
Network: demov3
Timestamped path or body: /nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a?ts=1689025512273
Token: Authorization: Bearer ES256K_En3WkxJQBgHuLazoZ9NYMkEWiLg9M3vfesccRtd3sdACMzfG3tvEhiA2Xtrqwr74WHqKtHRXJngvdBSEevRvhTxAf
```

Comment (elv-peter) [reply utilities/EluvioLiveCli.js]: not sure if this is valid YAML with the extra colons

**Pull request merged**